hub/src/server/drivers/diskDriver.ts

```diff
@@ -243,7 +243,16 @@ class DiskDriver implements DriverModel {
       contentTypeFilePath, 
       JSON.stringify({ 'content-type': contentType }), { mode: 0o600 })
```

Comment:
(Related to the comment below about storing etags as uuids in the metadata file)
This could be something like 
```js
JSON.stringify({ 
  'content-type': contentType,
  'etag': crypto.randomBytes(32).toString('hex')
})